### ISSUE TITLE
Does not detect control structures without curly braces

### ISSUE THREAD

**Issue by thled** [slevomat/coding-standard#1060]
It looks to me like if I do not use curly braces with control structures they do not count as one.

Example:

```php
public function foo(): bool
{
    if (true)
        return true;

    return false;
}
```

This gives the error: "Expected 1 lines before "return", found 0.".
If I use curly braces for the "if" it does not return the error.

Is this behavior intentional?

**Comment by kukulich** [comment]
Yes, we don’t support syntax without curly braces.

**Comment by thled** [comment]
Unfortunate.

Thanks for your fast reply!
